README.md

```diff
@@ -57,10 +66,10 @@ For other issues, please use the following repos:
 
 There are many .NET related projects on GitHub.
 
-- [.NET home repo](https://github.com/Microsoft/dotnet) - links to 100s of .NET projects, from Microsoft and the community.
-- [ASP.NET Core home](https://docs.microsoft.com/aspnet/core) - the best place to start learning about ASP.NET Core.
+* [.NET home repo](https://github.com/Microsoft/dotnet) - links to 100s of .NET projects, from Microsoft and the community.
+* [ASP.NET Core home](https://docs.microsoft.com/aspnet/core) - the best place to start learning about ASP.NET Core.
 
-This project has adopted the code of conduct defined by the [Contributor Covenant](https://contributor-covenant.org) to clarify expected behavior in our community. For more information, see the [.NET Foundation Code of Conduct](https://www.dotnetfoundation.org/code-of-conduct).
+This project has adopted the code of conduct defined by the [Contributor Covenant](https://contributor-covenant.org) to clarify expected behavior in our community. For more information, see the [.NET Foundation Code of Conduct](https://www.dotnetfoundation.org/code-of-conduct) and this repo's [Code of Conduct doc](CODE-OF-CONDUCT.md).
```

Comment:
CODE-OF-CONDUCT.md just contains a link to https://www.dotnetfoundation.org/code-of-conduct , so it looks redundant to mention it here.

Reply:
Which one would be the better option? I think it's pointless to have a file that is unreachable. So we could either get rid of it, or just point to it in the README, rather than explain it there.

Reply:
CODE-OF-CONDUCT.md is mandatory boilerplate: https://github.com/dotnet/org-policy/blob/main/doc/PR15.md